qutip/tests/core/test_numpy_backend.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import numpy
+from unittest.mock import Mock
+
+from qutip.core.numpy_backend import np
+from qutip import CoreOptions
+
+# Mocking JAX to demonstrate backend switching
+mock_jax = Mock()
+mock_np = numpy
+
+
+class TestNumpyBackend:
+    def test_getattr_numpy(self):
+        with CoreOptions(numpy_backend=mock_np):
+            assert np.sum([1, 2, 3]) == numpy.sum([1, 2, 3])
```

Comment:
Adding one strict assert, but otherwise good.
```suggestion
            assert np.sum([1, 2, 3]) == numpy.sum([1, 2, 3])
            assert np.sum is numpy.sum
```